static/templates/recipient_row.hbs

```diff
@@ -4,17 +4,18 @@
         <div class="message-header-contents">
             {{! stream link }}
             <a class="message_label_clickable narrows_by_recipient stream_label {{color_class}}"
-                style="background: {{background_color}}; border-left-color: {{background_color}};"
-                href="{{stream_url}}"
-                title="{{#tr}}Narrow to stream &quot;{display_recipient}&quot;{{/tr}}">
+              style="background: {{background_color}}; border-left-color: {{background_color}};"
+              href="{{stream_url}}"
+              title="{{#tr}}Narrow to stream &quot;{display_recipient}&quot;{{/tr}}">
                 {{! invite only lock }}
                 {{#if invite_only}}
-                <i class="fa fa-lock recipient-row-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
+                    <i class="fa fa-lock recipient-row-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
                 {{/if}}
                 {{#if is_web_public}}
-                <i class="fa fa-globe recipient-row-stream-icon" title="{{t 'This is a web public stream' }}" aria-label="{{t 'This is a web public stream' }}"></i>
+                    <i class="fa fa-globe recipient-row-stream-icon" title="{{t 'This is a web public stream' }}" aria-label="{{t 'This is a web public stream' }}"></i>
                 {{/if}}
-                {{display_recipient}}</a>
+                {{display_recipient}}
+            </a>
```

Comment:
This is a user-visible change. It appends a space to the stream label.

We might decide we’re okay with this change (though I don’t think it’s ideal), but it seems dangerous to let a linter automatically make a change that isn’t semantically a no-op.

Reply:
One quick thing to clarify is that the linter didn't make this change automatically; that was me.  But I made the change due to the linter not liking the original code.  I will discuss with Anders on chat.